test/monitoring/aws-kinesisanalytics/KinesisDataAnalyticsMonitoring.test.ts

```diff
@@ -18,6 +18,24 @@ test("snapshot test: no alarms", () => {
   expect(Template.fromStack(stack)).toMatchSnapshot();
 });
 
+test("snapshot test: full restart rate alarm", () => {
```

Comment:
Doesn't the addition in the following test already cover the same thing as this new test?